assets/js/edd-checkout-global.js

```diff
@@ -184,9 +184,12 @@ window.EDD_Checkout = (function($) {
 
 						} else {
 
-							if (!inputs.is('.card-address-2')) {
-								inputs.attr('required','required');
-							}
+							inputs.each( function () {
+								var $input = $( this );
+								if ( ! $input.is( '.card-name, .card-address, .card-address-2' ) ) {
```

Comment:
@robincornett The default implementation of `.card-address` can be required: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/master/includes/checkout/template.php#L353

It would still fall back to server validation but I'm not sure if we should add another exclusion here that hasn't explicitly caused an issue -- unless you're seeing similar behavior with this field on a gateway?

Reply:
With taxes enabled, I was getting the JS error on all three of those fields after a discount was applied, which is why I'd added it. That was with Stripe.

Reply:
@robincornett I see that now. Okay, I think this makes sense.